pycbc/vetoes/chisq.py

```diff
@@ -383,11 +383,103 @@ def values(self, corr, snrv, snr_norm, psd, indices, template):
                     rchisq[above] = chisq
             else:
                 rchisq = chisq
-
-            return rchisq, numpy.repeat(dof, len(indices))# dof * numpy.ones_like(indices)
+            dof = numpy.repeat(dof, len(indices))
+            return rchisq, dof
         else:
             return None, None
 
```

Comment:
It seems inconsistent to add this in the chisq.py file. In the past we've used one file per type of chi-squared (bank_chisq.py, cont_chisq.py) ... As this one is probably more different than those two, it feels weird to lump it in with Bruce chi-squared.